Cabal/doc/developing-packages.rst

```diff
@@ -1716,10 +1716,16 @@ system-dependent values for these fields.
     Cabal can make sure that specified programs are built and on the ``PATH`` before building the component in question.
     It will always do so for internal dependencies, and also do so for external dependencies when using Nix-style local builds.
 
+    :pkg-field:`build-tool-depends` was added in Cabal 2.0, and it will
+    be ignored (with a warning) with old versions of Cabal.  See
+    :pkg-field:`build-tools` for more information about backwards
+    compatibility.
+
 .. pkg-field:: build-tools: program list
    :deprecated:
 
-    Deprecated in favor of :pkg-field:`build-tool-depends`.
+    Deprecated in favor of :pkg-field:`build-tool-depends`, but see
+    below for backwards compatibility information.
```

Comment:
There is another deprecation notice I modeled this on. It would be cool if `pkg-field` itself did the linking, but that may be too much to ask (it is used in the defining heading too).